keps/sig-windows/2578-windows-conformance/README.md

```diff
@@ -0,0 +1,431 @@
+
+We propose the following definitions as the Windows conformance standard for Kubernetes clusters that are released after Kubernetes 1.23 and onwards.  This specification is to be updated as new features are added or removed from the Kubernetes API itself.
+
+### User Stories (Optional)
+
+#### Story 1
+
+As a Windows Application developer I want to verify that the features I rely on in Kubernetes are supported on my specific Windows Kubernetes cluster.
+
+#### Story 2
+
+As a Kubernetes vendor I want to evaluate the completeness of my Kubernetes support matrix in context of Windows supported features in the Kubernetes API.
+
+#### Story 3
+
+As a Kubernetes developer I want to immediately know whether or not the features I want to add will affect the Windows support matrix. If there is a confirmed or potential impact I want to be able to rapidly convey this in a canonical way to the broader community.
+
+#### Story 4
+
+As an IT administrator, I want to know whether my current version of Kubernetes supports ActiveDirectory or other Windows features.
+
+### Risks and Mitigations
+
+<!--
+What are the risks of this proposal, and how do we mitigate? Think broadly.
+For example, consider both security and how this will impact the larger
+Kubernetes ecosystem.
+
+How will security be reviewed, and by whom?
+
+How will UX be reviewed, and by whom?
+
+Consider including folks who also work outside the SIG or subproject.
+-->
+
+## Design Details
+
+### Specification of Windows Operational Readiness (Kubernetes 1.23)
+
+- In order to qualify as a Windows test, the pods that demonstrate functionality for a given attribute of a cluster must be running on a `pause` image. 
+
+Additionally, this image must be scheduled to an operating system that meets the following requirements:
+
+- Windows Build Versions:
+  - Windows Server 2019 (LTSC)
+  - Windows Server 2022 (LTSC)
+
+- Installed KBs:
+  <!-- - Needs further discussion/investigation -->
+
+- Windows Hosts and containers must be running a patch level that is no older than 3 months
+
+- All tests in the Core Windows Operational Readiness suite depend on `pause`, and no other images.
+- All tests should use the `Pod.OS` specification field as the *only* required scheduling directive for pods, once this field is made GA.  Up until this time, the way tests schedule Windows pods can use RuntimeClasses, Labels, or taints in any way which "works" to implement the other requirements.
+
+#### Basic Networking
```

Comment:
These are lists of functionally, has there been any mapping to the current Conformance tests that run on windows and Windows specific tests to find gaps in functionality that isn't tested for windows?

Reply:
yeah, these are reverse engineered from the tests we already have,  ive manually confirmed we have tags for these.